data/controllers.yml

```diff
@@ -15,7 +15,7 @@ questions:
             - {value: return $this->getContainer()->get('service.name'), correct: false}
             - {value: return $this->container->get('service.name'),      correct: true}
     -
-        question: 'Which class must be extended by your controllers?'
+        question: 'Which class may be extended by your controllers?'
         answers:
             - {value: Symfony\Bundle\FrameworkBundle\Controller,    correct: true}
```

Comment:
This is not true. The FQCN is `Symfony\Bundle\FrameworkBundle\Controller\Controller`.